packages/wp-now/README-new.md

```diff
@@ -0,0 +1,140 @@
+
+Once installed, you can use it like so:
+
+```bash
+cd wordpress-plugin-or-theme
+wp-now start
+```
+
+Use the `--php=<version>` and `--wp=<version>` arguments to switch to different versions on the fly:
+
+```bash
+wp-now start --wp=5.9 --php=7.4
+```
+
+### Automatic Modes
+
+`wp-now` automatically operates in a few different modes. The selected mode depends on the directory in which it is executed:
+
+-   `plugin` or `theme`: Loads the plugin or theme files into a virtual filesytem with WordPress and a SQLite-based database. Everything (including WordPress core files, te database, `wp-config.php`, etc.) is stored in the user's home directory. The latest version of WordPress will be used, unless the `--wp=<version>` argument is provided.
+-   `wp-content`: Loads the project files as a `wp-content` folder within a virtual filesystem. WordPress core files and `wp-config.php` remain virtualized, but the SQLite database will be written to the project directory. The latest version of WordPress will be used, unless the `--wp=<version>` argument is provided.
+-   `wordpress`: Runs the directory as a WordPress installation when WordPress files are detected. An existing `wp-config.php` file will be used if it exists; if it doesn't exist, it will be created along with a SQLite database.
+-   `wordpress-develop`: Same as `wordpress` mode, except the `build` directory is served as the web root.
+-   `index`: Starts a PHP webserver in the working directory and simply passes requests to `index.php`.
+
+### Arguments
+
+`wp-now start` currently supports the following arguments:
+
+-   `--path`: the path to the PHP file or WordPress project to use. If not provided, it will use the current working directory;
+-   `--php`: the version of PHP to use. This is optional and if not provided, it will use a default version which is `8.0`(example usage: `--php=7.4`);
+-   `--port`: the port number on which the server will listen. This is optional and if not provided, it will pick an open port number automatically. The default port number is set to `8881`(example of usage: `--port=3000`);
+-   `--wp`: the version of WordPress to use. This is optional and if not provided, it will use a default version. The default version is set to the [latest WordPress version](https://wordpress.org/download/releases/)(example usage: `--wp=5.8`)
+
+## Technical Details
+
+-   The `~/.wp-now` home directory is used to store the WP versions and the `wp-content` folders for projects using theme and plugin mode. The path to `wp-content` directory for the `plugin` and `theme` modes is `~/.wp-now/wp-content/${projectName}`.
+-   For the database setup, `wp-now` is using [Sqlite database integration plugin](https://wordpress.org/plugins/sqlite-database-integration/). The path to Sqlite database is ` ~/.wp-now/wp-content/${projectName}/database/.ht.sqlite`
```

Comment:
It's not really `${projectName}` but `${projectName}-${directoryHash}` where directoryHash is sha1 hash calculated from the full path to the project. Should we clarify this somehow?

Reply:
Added `${directoryHash}` mention with a5c4a6a8